apps/crn-server/src/controllers/user.controller.ts

```diff
@@ -32,9 +32,7 @@ export default class UserController {
   }
 
   async fetch(options: FetchUsersOptions): Promise<ListUserResponse> {
```

Comment:
pretty sure that the fetch method in the controller should change too?

Reply:
But I think it's already with the right type since https://github.com/yldio/asap-hub/blob/asap-62-users-2/packages/model/src/user.ts#L194

```
export type ListUserResponse = ListResponse<UserListItemResponse>
```